pkg/report/output/privacy/formatter.go

```diff
@@ -0,0 +1,54 @@
+package privacy
+
+import (
+	"fmt"
+
+	"github.com/bearer/bearer/pkg/commands/process/settings"
+	"github.com/bearer/bearer/pkg/flag"
+	"github.com/bearer/bearer/pkg/report/output/html"
+	outputtypes "github.com/bearer/bearer/pkg/report/output/types"
+	outputhandler "github.com/bearer/bearer/pkg/util/output"
+)
+
+type Formatter struct {
+	ReportData *outputtypes.ReportData
+	Config     settings.Config
+}
+
+func NewFormatter(reportData *outputtypes.ReportData, config settings.Config) *Formatter {
+	return &Formatter{
+		ReportData: reportData,
+		Config:     config,
+	}
+}
+
+func (f Formatter) Format(format string) (output *string, err error) {
+	switch format {
+	case flag.FormatEmpty:
```

Comment:
It feels like we should default to `json` and add a new `flag.FormatCSV` option. Not being able to get this `csv` other than not specifying a type feels like a smell to me

Reply:
I think the default format for privacy report should remain CSV (otherwise it's quite a big change in functionality), but you're right, we could add a flag for this and have something like:

```
case flag.FormatEmpty, flag.FormatCSV:
```

Reply:
Added this in a separate PR here https://github.com/Bearer/bearer/pull/1208 - so that this PR can remain refactoring only (no functionality changes)